src/driver/AmdCompiler.cpp

```diff
@@ -79,6 +81,9 @@ using namespace llvm::object;
 using namespace clang;
 using namespace clang::driver;
 
+static cl::opt<bool>
+Verbose("v", cl::ZeroOrMore, cl::desc("Print information about actions taken"), cl::init(false));
```

Comment:
-v is option used by clang, llvm-link and others. The idea was to give a way to support the existing LLVM/Clang options in RocM OpenCL driver.